Detectors/AOD/src/aod-producer-workflow.cxx

```diff
@@ -36,7 +36,7 @@ void customize(std::vector<ConfigParamSpec>& workflowOptions)
     {"disable-root-output", o2::framework::VariantType::Bool, false, {"disable root-files output writer"}},
     {"disable-mc", o2::framework::VariantType::Bool, false, {"disable MC propagation"}},
     {"disable-secondary-vertices", o2::framework::VariantType::Bool, false, {"disable filling secondary vertices"}},
-    {"disable-strangeness-tracker", o2::framework::VariantType::Bool, false, {"disable filling strangeness tracking"}},
+    {"disable-strangeness-tracking", o2::framework::VariantType::Bool, false, {"disable filling strangeness tracking"}},
```

Comment:
Please don't modify options, even if it sounds better it will break dependent scripts.